Makefile

```diff
@@ -31,12 +32,8 @@ lint: $(golangci_lint_path)
 
 .PHONY: format
 format:
-	@find . -type f -name '*.go' | xargs gofmt -s -w
-	@for f in `find . -name '*.go'`; do \
-	    awk '/^import \($$/,/^\)$$/{if($$0=="")next}{print}' $$f > /tmp/fmt; \
```

Comment:
Does gofumpt handle the import grouping that I think this was for?

Reply:
Though if not, while on the note of swapping formatting tools, can consider gosimports

https://github.com/rinchsan/gosimports

Reply:
I think gofumpt mainly groups stdlib imports, so I think you are right it wouldn't be able to correct dependencies vs our imports.